dvc/output.py

```diff
@@ -1138,6 +1187,7 @@ def is_plot(self) -> bool:
     Output.PARAM_PLOT: bool,
     Output.PARAM_PERSIST: bool,
     Output.PARAM_CHECKPOINT: bool,
+    Output.PARAM_TREE: [TREE_ENTRY_SCHEMA],
```

Comment:
I’d call it `files` instead.

Reply:
went with `files`, it probably will make more sense to users since we already use `nfiles`